zookeeper-client/zookeeper-client-c/tests/TestReconfig.cc

```diff
@@ -499,7 +499,7 @@ class Zookeeper_reconfig : public CPPUNIT_NS::TestFixture
             found = seen.find(next.str());
             CPPUNIT_ASSERT_MESSAGE(next.str() + " in seen list",
                                    found == string::npos);
-            seen += found + ", ";
+            seen += next.str() + ", ";
```

Comment:
I think this is a bug but not a simple "nice to have". I experimented that this `+=` will convert `found` from integer to char.  cc @ztzg

Reply:
@ztzg could you help to check this pull request, thanks :)